api/v1alpha1/flowcollector_types.go

```diff
@@ -144,16 +156,21 @@ type FlowCollectorHPA struct {
 	Metrics []ascv2.MetricSpec `json:"metrics"`
 }
 
-// FlowCollectorLoki defines the desired state for FlowCollector's Loki client
+// FlowCollectorLoki defines the desired state for FlowCollector's Loki configuration
 type FlowCollectorLoki struct {
+	// InstanceSpec is the Spec section of LokiStack instance
```

Comment:
Could you add something along these lines: "Experimental feature. If provided, the Loki operator will be automatically installed. At this moment, the Loki operator itself is experimental and hasn't been officially released by Grafana." ?
(or said differently, but you get the point...)